MobileCenterCrashes/MobileCenterCrashes/Internals/MSWrapperExceptionManager.h

```diff
@@ -1,111 +1,42 @@
 #import <Foundation/Foundation.h>
 
-@class MSException;
-
-/**
- * This is required for Wrapper SDKs that need to install their own
- * signal handlers for certain signals.
- */
-@protocol MSWrapperCrashesInitializationDelegate <NSObject>
-
-/**
- * Implement this function to override the default behavior for
- * setting up the crash handlers (i.e., configuring PLCrashReporter)
+@class MSWrapperException;
+
+/**
+ * This class helps wrapper SDKs augment crash reports with additional data.
+ *
+ * HOW IT WORKS:
+ * 1. Application is crashing from a wrapper SDK, but before propagating
+ *    crash to the Objective-C code, it calls "saveWrapperException"
+ *    which saves the MSWrapperException to a file called 
+ *    "last_saved_wrapper_exception".
+ * 2. On startup, the Objective-C SDK must find that file if it exists,
```

Comment:
I can change it. I know it sounds wrong; I was just trying to very clearly differentiate the underlying SDK from the wrapper iOS SDKs